test/spec/spectest.cpp

```diff
@@ -562,18 +569,27 @@ void SpecTest::run(std::string_view Proposal, std::string_view UnitName) {
   };
 
   // Command processing. Return true for expected result.
-  auto RunCommand = [&](const rapidjson::Value &Cmd) {
-    // Line number in wast: Cmd["line"].Get<uint32_t>()
-    if (const auto Type = Cmd.FindMember("type"s); Type != Cmd.MemberEnd()) {
-      switch (resolveCommand(Type->value.Get<std::string>())) {
+  auto RunCommand = [&](const simdjson::dom::object &Cmd) {
+    std::string_view TypeField;
+
+    if (!Cmd["type"].get(TypeField)) {
+      switch (resolveCommand(TypeField)) {
       case SpecTest::CommandID::Module: {
-        const auto FileName = (TestsuiteRoot / Proposal / UnitName /
-                               Cmd["filename"].Get<std::string>())
-                                  .u8string();
-        const uint64_t LineNumber = Cmd["line"].Get<uint64_t>();
-        if (const auto Name = Cmd.FindMember("name"); Name != Cmd.MemberEnd()) {
+        std::string_view Name = Cmd["filename"];
+        const auto FileName =
+            (TestsuiteRoot / Proposal / UnitName / Name).u8string();
```

Comment:
Why not use `Cmd["filename"]` instead?

Reply:
Then it returns `simdjson::result<std::string_view>`, which presents the same problem I mentioned [above](https://github.com/WasmEdge/WasmEdge/pull/2659/files#r1270234563)